### PR TITLE
Composer improvements

### PR DESCRIPTION
**Pull request by soullivaneuh**
- New requirement constraint notations
- Remove minimum-stability option
- Admin bundle ^3.0
- sllh/php-cs-fixer-styleci-bridge (for dev-kit)

### CODE REVIEW

**Comment by soullivaneuh** [comment]
Note: I also removed SF 3 tests from Travis. Your bundle is not compatible.

**Comment by soullivaneuh** [comment]
@greg0ire I let you merge this and https://github.com/sonata-project/dev-kit/pull/99 if it's OK for you.

File: composer.json [hunk 1]
```diff
@@ -17,15 +17,16 @@
...
+        "sonata-project/admin-bundle": "^3.0"
     },
     "require-dev": {
-        "matthiasnoback/symfony-dependency-injection-test": "~0.7.1",
-        "matthiasnoback/symfony-config-test": "~0.4.0|~1.0",
-        "symfony/phpunit-bridge": "~2.7|~3.0"
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7.1",
+        "matthiasnoback/symfony-config-test": "^0.4.0 || ^1.0",
+        "symfony/phpunit-bridge": "^2.7 || ^3.0",
+        "sllh/php-cs-fixer-styleci-bridge": "^2.0"
```
Comment (greg0ire) [inline composer.json]: Please sort dependencies

**Comment by soullivaneuh** [comment]
@greg0ire fixed.

**Comment by greg0ire** [comment]
Shit, I should not have squashed! Oh well... thanks @Soullivaneuh

**Pull request merged**